openmetadata-ui/src/main/resources/ui/src/components/Settings/Services/Ingestion/Ingestion.component.tsx

```diff
@@ -264,7 +264,6 @@ const Ingestion: React.FC<IngestionProps> = ({
       <EntityDeleteModal
         entityName={deleteSelection.name}
         entityType={t('label.ingestion-lowercase')}
-        loadingState={deleteSelection.state}
```

Comment:
Can we also remove `deleteSelection`

Reply:
we will need this in order to handle the status of pipeline deletion